src/test/java/com/celements/parents/DocumentParentsListerTest.java

```diff
@@ -0,0 +1,190 @@
+package com.celements.parents;
+
+import static org.easymock.EasyMock.*;
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.xwiki.model.EntityType;
+import org.xwiki.model.reference.DocumentReference;
+
+import com.celements.common.test.AbstractBridgedComponentTestCase;
+import com.xpn.xwiki.XWiki;
+import com.xpn.xwiki.XWikiContext;
+import com.xpn.xwiki.XWikiException;
+import com.xpn.xwiki.doc.XWikiDocument;
+import com.xpn.xwiki.web.Utils;
+
+public class DocumentParentsListerTest extends AbstractBridgedComponentTestCase {
+
+  private XWikiContext context;
+  private XWiki xwiki;
+  private DocumentParentsLister docParentsLister;
+  private Map<String, IDocParentProviderRole> docParentProviderMapBackup;
+
+  @Before
+  public void setUp_WebUtilsServiceTest() throws Exception {
+    context = getContext();
+    xwiki = getWikiMock();
+    docParentsLister = (DocumentParentsLister) Utils.getComponent(
+        IDocumentParentsListerRole.class);
```

Comment:
no, getComponentManager() provides a ComponentManager object which deferes in unstable-2.0 branch and dev branch because of  lookup<->getInstance